.circleci/config.yml

```diff
@@ -435,6 +435,38 @@ jobs:
           name: Setup Elasticsearch Index Settings
           command: |
             ./web-api/setup-elasticsearch-index.sh $ENV
+      - run:
```

Comment:
Do you think we could run these new steps in its own job after the `deploy` job?  If so, we should move these new steps to their own job after `deploy` so they can have their own [branch filter](https://support.circleci.com/hc/en-us/articles/115015953868-Filter-workflows-by-branch-).  That way you don't need any of the `if [ "${CIRCLE_BRANCH}" != "prod" ]; then` statements at all.

Specifically, you'd use an `ignore` statement that lists `prod`.  [Reference](https://circleci.com/docs/2.0/configuration-reference/#filters).

Reply:
Oh hmmm.  Then I recommend making two additional jobs.  One for the adding the users (with the branch filter), and one for the smoke tests.

But I'm OK with your current approach.  I would just prefer doing it through two additional jobs.  Your decision.

Reply:
I think it depends on the problem we are trying to solve here. If we are trying to only solve the issue with experimental environments I think seeding test users could go in a step after the deploy - I say that because the `Deploy - Web API - Smoke Tests` step talks directly to cognito so we don't have to have users seeded in dynamo like we do for the cypress smoketests that occur later.

Reply:
I don't have an opinion either way! Just wanted to clarify 😄